### PR TITLE
Localhost: always deploy ArgentAccount

### PR DESCRIPTION
**Pull request by gergold**

### CODE REVIEW

File: packages/extension/src/background/index.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import ArgentAccountCompiledContract from "!!raw-loader!../contracts/ArgentAccount.txt"
```
Comment (janek26) [inline packages/extension/src/background/index.ts:1]: 🤮
hopefully starknet will fix this soon

File: packages/extension/src/background/wallet.ts [hunk 2]
```diff
@@ -154,11 +156,27 @@ export class Wallet {
     this.accounts.push(account)
 
     await this.writeBackup()
-    await this.selectAccount(deployTransaction.address)
+    await this.selectAccount(account.address)
 
     return { account, txHash: initTransaction.transaction_hash }
```
Comment (janek26) [inline packages/extension/src/background/wallet.ts:161]: does it show as 2 transactions in the ui? or are we just dropping the implementation transaction from the transaction tracker?

**Review by janek26 — APPROVED** [review]
love the tests. Few nitpicks, but ok overall

File: packages/extension/src/background/wallet.ts (hunk 2 above)
Comment (gergold) [reply packages/extension/src/background/wallet.ts:161]: yes, we just track the second one... until Starknet implements the factory pattern and we can bundle them into a single tx

**Pull request merged**